### ISSUE TITLE
JSON decoding for fetching deployments fail

### ISSUE THREAD

**Issue by jonathanballs** [telepresenceio/telepresence#989]
**Versions**
kubectl 1.14.0
telepresence 0.97

**What happened**
Ran `telepresence` and it fails immediately with the following error:
```
╭─jonathan@jonathanxps ~  
╰─➤  telepresence                                                           1 ↵
T: Invoking sudo. Please enter your sudo password.
[sudo] password for jonathan: 
T: Starting proxy with method 'vpn-tcp', which has the following limitations: 
T: All processes are affected, only one telepresence can run per machine, and 
T: you can't use other VPNs. You may need to add cloud hosts and headless 
T: services with --also-proxy. For a full list of method limitations see 
T: https://telepresence.io/reference/methods.html
T: Volumes are rooted at $TELEPRESENCE_ROOT. See 
T: https://telepresence.io/howto/volumes.html for details.
T: Starting network proxy to cluster using new Deployment 
T: telepresence-1555078708-4893398-22244

Looks like there's a bug in our code. Sorry about that!

Traceback (most recent call last):
  File "/usr/bin/telepresence/telepresence/cli.py", line 130, in crash_reporting
    yield
  File "/usr/bin/telepresence/telepresence/main.py", line 60, in main
    remote_info = start_proxy(runner)
  File "/usr/bin/telepresence/telepresence/proxy/__init__.py", line 104, in start_proxy
    run_id=run_id,
  File "/usr/bin/telepresence/telepresence/proxy/remote.py", line 156, in get_remote_info
    runner, deployment_name, deployment_type, run_id=run_id
  File "/usr/bin/telepresence/telepresence/proxy/remote.py", line 101, in get_deployment_json
    stderr=STDOUT
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)


Here are the last few lines of the logfile (see /home/jonathan/telepresence.log for the complete logs):

   9.5 TEL | Scout info: {'latest_version': '0.98', 'application': 'telepresence', 'notices': []}
   9.6 TEL | BEGIN SPAN deployment.py:61(create_new_deployment)
   9.6 >>> | Starting network proxy to cluster using new Deployment telepresence-1555078708-4893398-22244
   9.6 TEL | [10] Capturing: kubectl --context do-lon1-demo-nanowire-io --namespace default delete --ignore-not-found svc,deploy --selector=telepresence=5a48bc9d3bbd4bb5afb5cbf9495a29a4
   9.7 TEL | [11] Capturing: kubectl --context do-lon1-demo-nanowire-io --namespace default run --restart=Always --limits=cpu=100m,memory=256Mi --requests=cpu=25m,memory=64Mi telepresence-1555078708-4893398-22244 --image=datawire/telepresence-k8s:0.97 --labels=telepresence=5a48bc9d3bbd4bb5afb5cbf9495a29a4
   9.9  11 | kubectl run --generator=deployment/apps.v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
   9.9  11 | deployment.apps/telepresence-1555078708-4893398-22244 created
   9.9 TEL | END SPAN deployment.py:61(create_new_deployment)    0.4s
   9.9 TEL | BEGIN SPAN remote.py:154(get_remote_info)
   9.9 TEL | BEGIN SPAN remote.py:78(get_deployment_json)
   9.9 TEL | [12] Capturing: kubectl --context do-lon1-demo-nanowire-io --namespace default get deployment -o json --export --selector=telepresence=5a48bc9d3bbd4bb5afb5cbf9495a29a4
  10.1 TEL | END SPAN remote.py:78(get_deployment_json)    0.2s

Would you like to file an issue in our issue tracker? You'll be able to review and edit before anything is posted to the public. We'd really appreciate the help improving our product. [Y/n]: 
```

The cause seems to be the command `kubectl --context do-lon1-demo-nanowire-io --namespace default get deployment -o json --export --selector=telepresence=5a48bc9d3bbd4bb5afb5cbf9495a29a4` which when I run on my machine returns the output (truncated):

```
╭─jonathan@jonathanxps ~  
╰─➤  kubectl --context do-lon1-demo-nanowire-io --namespace default get deployment -o json --export --selector=telepresence=5a48bc9d3bbd4bb5afb5cbf9495a29a4
Flag --export has been deprecated, This flag is deprecated and will be removed in future.
{
    "apiVersion": "v1",
    "items": [
...
```

Essentially the deprecation warning is causing the JSON parser to fail.

**Comment by richarddli** [comment]
This is fixed on Telepresence 0.98.

**Comment by jonathanballs** [comment]
Thank you for your very quick response!
